src/azul/service/manifest_service.py

```diff
@@ -592,12 +596,19 @@ def _get_entities(self, path: str, doc: JSON) -> List[JSON]:
         entities = d.get(path[-1], [])
         return entities
 
-    def _file_url(self, file: JSON) -> Optional[str]:
+    def _repository_file_url(self, file: JSON) -> Optional[str]:
         replica = 'gcp'  # BDBag is for Terra and Terra is GCP
         return self.repository_plugin.direct_file_url(file_uuid=file['uuid'],
                                                       file_version=file['version'],
                                                       replica=replica)
 
+    def _azul_file_url(self, file: JSON, args: Mapping = frozendict()) -> str:
```

Comment:
This should make use of the `FileUrlFunc` abstraction. We don't want services to be married to URL specifics. Can you add a ticket+FIXME, please?